javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,9 @@
+let dogyearOfBirth = prompt("Please enetr your dog birth year");
+let dogyearOfBirthNumber = Number(dogyearOfBirth);
+let dogyearFuture = prompt("Please enter the future year");
+let dogyearFutureNumber = Number(dogyearFuture);
+let humanYearAge = dogyearFutureNumber - dogyearOfBirthNumber;
+let dogYearAge = 10 * humanYearAge;
+let shouldShowResultInDogYears = prompt("Do you want to calculate in dog years?")
+if (shouldShowResultInDogYears == "yes") { console.log("Your dog will be " + dogYearAge + " dog years old in " + dogyearFuture + "."); }
+console.log("Your dog will be " + humanYearAge + " human years old in " + dogyearFuture + ".")
```

Comment:
here because we want to console.log the dog year or the human year and not both together we should do it with if and else condition, and please try to follow the way I wrote my if and else and the way I indented so it can be more readable and also easier to understand.
```suggestion
if (shouldShowResultInDogYears == "yes") {
       console.log("Your dog will be " + dogYearAge + " dog years old in " + dogyearFuture + "."); 
} else {
       console.log("Your dog will be " + humanYearAge + " human years old in " + dogyearFuture + ".")
}
```